datasets/superb/README.md

```diff
@@ -260,17 +303,20 @@ An example from each split looks like:
 #### ks
 
 - `file` (`string`): Path to the WAV audio file.
-- `label` (`string`): Label of the spoken command.
+- `label` (`ClassLabel`): Label of the spoken command.
 
 #### qbe
 
 [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
 
-
 #### ic
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-
+- `file` (`string`): Path to the WAV audio file.
+- `speaker_id` (`string`): ID of the speaker.
+- `text` (`string`): Transcription of the spoken command.
+- `action` (`ClassLabel`): Label of the command's action.
+- `object` (`ClassLabel`): Label of the command's object.
+- `location` (`ClassLabel`): Label of the command's location.
```

Comment:
Maybe we can have the mapping between the label ids (int) and the label names (str) here ?